setup.py

```diff
@@ -1,31 +1,33 @@
 #!/usr/bin/env python
-"""
-Package metadata for openedx_certificates.
-"""
+"""Package metadata for openedx_certificates."""
```

Comment:
Do we still need `setup.py`? Can't we declare all this using `pyproject.toml` which is the replacement, and which is more efficient since it doesn't have to be run as code before it can be used?